app/server-control/directives/kvm-console.js

```diff
@@ -30,11 +30,18 @@ window.angular && (function(angular) {
                 return false;
               };
 
+              var flag = 0;
               function connected(e) {
                 $log.debug('RFB Connected');
+                flag = 1;
               }
               function disconnected(e) {
                 $log.debug('RFB disconnected');
+                if (flag ==0)
```

Comment:
flag == 0 
Can you run: https://github.com/ibm-openbmc/phosphor-webui/blob/master/format-code.sh

Reply:
I will try, thanks for your suggestion.